packages/api-core/src/resources/filesDelivery.js

```diff
@@ -0,0 +1,39 @@
+import AvMicroservice from '../ms';
+
+export default class AvFilesDelivery extends AvMicroservice {
+  constructor({ http, promise, merge, config }) {
+    const options = Object.assign(
```

Comment:
Why not use `merge` to better handle deep merging (such as if the user wants to add a custom header without needing to specify `'Content-Type': 'application/json'`)

Reply:
i like this idea. will open ticket.